app/konnect/dev-portal/access-and-approval/manage-teams.md

```diff
@@ -0,0 +1,230 @@
+
+{% navtabs %}
+{% navtab Konnect UI %}
+1. From the navigation menu, open {% konnect_icon dev-portal %} **Dev Portal**.
+1. Click **Teams** in the sidebar and then click **Add team**.
+1. Enter the following:
+    * **Team name:** Delivery Partners
+    * **Description:** Team of vetted delivery partners that can view and consume APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Classic Pizzas** product from the dropdown menu and select **API Consumer** and **API Viewer** from the **Add roles** dropdown menu.
+1. Click **Teams** in the sidebar and click **New Team** again and enter the following:
+    * **Team name:** New Partners
+    * **Description:** Team of unvetted partners that can only view APIs
+1. On the **Developers** tab of your new team, click **Add developer** and select a developer to add to the team.
+1. On the **Products** tab of your new team, click **Add Role**.
+1. Select the **Classic Pizzas** product from the dropdown menu and select **API Viewer** from the **Add roles** dropdown menu. 
+{% endnavtab %}
+{% navtab API %}
+{:.note}
+> **Note:** Make sure you replace the placeholders in the following API requests with information specific to your environment. 
+
+1. Create the `Delivery Partners` team:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/portals/<portal-id>/teams \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "name": "Delivery Partners",
+    "description": "Team of vetted delivery partners that can view and consume APIs"
+  }'
+```
+1. Create the `New Partners` team:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/portals/<portal-id>/teams \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --data '{
+    "name": "New Partners",
+    "description": "Team of unvetted partners that can only view APIs"
+  }'
+```
+1. You can verify that the teams have been created by making a GET request to the teams endpoint:
+```bash
+curl --request GET \
+  --url https://<region>.api.konghq.com/v2/portals/<portal-id>/teams \
+  --header 'Authorization: Bearer <personal-access-token>'
+```
+1. Now you can assign roles to your teams. To do so, you must make a `POST` request to the team roles endpoint. The following example shows how to assign the `API Viewer` and `API Consumer` roles to the `Delivery Partners` created in the previous section for your classic pizza API product:
+```bash
+curl --request POST \
+  --url https://<region>.api.konghq.com/v2/portals/<portal-id>/teams/<team-id>/assigned-roles \
+  --header 'Authorization: Bearer <personal-access-token>' \
+  --header 'Content-Type: application/json' \
+  --data '{
```

Comment:
I think it might be this: 

`--data '{  "role_name": "API Viewer", "API Consumer",  "entity_id": "18ee2573-dec0-4b83-be99-fa7700bcdc61",  "entity_type_name": "Services",  "entity_region": "us"}'`

looks like region is required. https://developer.konghq.com/spec/2dad627f-7269-40db-ab14-01264379cec7/0ecb66fc-0049-414a-a1f9-f29e8a02c696#/Team%20Roles/portal-teams-assign-role